tdrs-backend/tdpservice/parsers/schema_defs/tanf/t5.py

```diff
@@ -15,66 +16,129 @@
+                  result_field='PARENT_MINOR_CHILD', result_function=validators.isInLimits(1, 3)
+                  ),
+              validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
+                  result_field='EDUCATION_LEVEL', result_function=validators.or_validators(
+                      validators.isInStringRange(1, 16, 2),
+                      validators.isInStringRange(98, 99)
+                      )
+                  ),
+              validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.matches(1),
+                  result_field='CITIZENSHIP_STATUS', result_function=validators.isInLimits(1, 2)
+                  ),
+              validators.validate__FAM_AFF__HOH__Count_Fed_Time(),
+              validators.if_then_validator(
+                  condition_field='DATE_OF_BIRTH', condition_function=validators.olderThan(18),
+                  result_field='REC_OASDI_INSURANCE', result_function=validators.isInLimits(1, 2)
+                  ),
+              validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.matches(1),
+                  result_field='REC_FEDERAL_DISABILITY', result_function=validators.isInLimits(1, 2)
+                  ),
+                ],
           fields=[
               Field(item="0", name='RecordType', type='string', startIndex=0, endIndex=2,
                     required=True, validators=[]),
               Field(item="4", name='RPT_MONTH_YEAR', type='number', startIndex=2, endIndex=8,
-                    required=True, validators=[]),
+                    required=True, validators=[validators.dateYearIsLargerThan(1998),
+                                               validators.dateMonthIsValid()]),
               Field(item="6", name='CASE_NUMBER', type='string', startIndex=8, endIndex=19,
-                    required=True, validators=[]),
+                    required=True, validators=[validators.isAlphaNumeric(),
+                                               validators.notEmpty()]),
               Field(item="14", name='FAMILY_AFFILIATION', type='number', startIndex=19, endIndex=20,
-                    required=True, validators=[]),
+                    required=True, validators=[validators.isInLimits(1, 5)]),
               Field(item="15", name='DATE_OF_BIRTH', type='number', startIndex=20, endIndex=28,
-                    required=True, validators=[]),
-              EncryptedField(decryption_func=tanf_ssn_decryption_func, item="16", name='SSN', type='string',
-                             startIndex=28, endIndex=37, required=True, validators=[]),
+                    required=True, validators=[validators.isLargerThan(0)]),
+              TransformField(transform_func=tanf_ssn_decryption_func, item="16", name='SSN', type='string',
```

Comment:
@elipe17 we are seeing evidence of SSNs being decrypted when encryption indicator is not `E`. see below:

![2633p1](https://github.com/raft-tech/TANF-app/assets/63075587/d5541513-0bfe-4227-b579-8b0fcd635468)

Reply:
@ADPennington Could you try redeploying? I am not able to reproduce this locally. Screenshots below of what I used to test. Could you send me your test file if the issue persists to see if I can reproduce the issue?
![Screenshot 2023-08-28 at 3 34 24 PM](https://github.com/raft-tech/TANF-app/assets/125676261/42b709f3-cacf-4e8f-b64e-7d5256942320)

Reply:
@ADPennington update. I redeployed to qasp environment and performed the same test and saw the same result. Could you try again and see if that was the issue for you too?
![Screenshot 2023-08-28 at 4 21 52 PM](https://github.com/raft-tech/TANF-app/assets/125676261/89e79e55-13fb-465b-b8d4-6118baeb3110)

Reply:
@elipe17 still seeing it
![2633p3](https://github.com/raft-tech/TANF-app/assets/63075587/b2ca5c46-af29-4050-aa0f-6c0d299c7d85)

Reply:
@ADPennington could you share your datafile with me so that I can debug with it?